src/server/api/routes/users.router.js

```diff
@@ -63,4 +63,73 @@ router.get('/:id', (req, res, next) => {
     .catch(next);
 });
 
+/**
+ * @swagger
+ * /users:
+ *  post:
+ *    tags:
+ *    - Users
+ *    summary: Save user information
+ *    description:
+ *      Will return status of the operation true / false
+ *    produces: application/json
+ *    parameters:
+ *     - in: path
+ *       name: user
+ *       schema:
+ *         type: json
+ *         required: true
+ *         description: user json object
+ *
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.post('/', (req, res, next) => {
+  usersController
+    .saveUser(req.body)
+    .then((result) => res.json(result))
+    .catch(next);
+});
+
+/**
+ * @swagger
+ * /users/{id}:
+ *  put:
+ *    tags:
+ *    - Users
+ *    summary: Update user information by id
+ *    description:
+ *      Will return status of the operation true / false
+ *    produces: application/json
+ *    parameters:
+ *     - in: path
+ *       name: id
+ *       schema:
+ *         type: integer
+ *         required: true
+ *         description: user id
+ *     - in: path
+ *       name: user
+ *       schema:
+ *         type: json
+ *         required: true
+ *         description: user json object
+ *
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.put('/:id', (req, res, next) => {
```

Comment:
Why did you decide to use PUT?
PUT requires to send all info from user while, and if  someone made a mistake it will not overwrite the previouse user but create a new one like POST.
PATCH is better because you can send just changed fields.
We aleady have patch for delivery info - pls merge develop and solve conflicts

Reply:
I have changed it, thanks